README.md

```diff
@@ -422,6 +422,38 @@ Expiring certificate:
       }
     }
 
+### IPACAChainExpirationCheck
+
+Load the CA chain from /etc/ipa/ca.crt and test each one for expiration. This test is designed to ensure that the entire CA chain for all certificates is validated. For example, if the web or LDAP certificates have been replaced then the CA chain for those certs will reside in /etc/ipa/ca.crt. This includes an IPA CA signed by an external authority.
+
+Expiring certificate:
+
+    {
+      "source": "ipahealthcheck.ipa.certs",
+      "check": "IPACAChainExpirationCheck",
+      "result": "WARNING",
```

Comment:
I'm not too keen on having that as a WARNING, ERROR makes more sense to me.

Reply:
It emits a warning when the cert is within the expiration days but not yet expired period.

Reply:
Normally the cluster should have enough monitoring so that it's renewed before healthcheck catches it? Being within the expiration days / not expired period is already an error then. But I'm not sure it matters much.